### PR TITLE
update stack-exact.yaml

### PR DESCRIPTION
**Pull request by shayne-fletcher**
ghc-9.2.2 resolver & the most recent compatible packages

### CODE REVIEW

File: stack-exact.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # You can add --resolver xxx to the above command to have the effect
 # of overriding the choice of compiler if desired.
 
-resolver: ghc-9.2.4
+resolver: ghc-9.2.2
```
Comment (akrmn) [inline stack-exact.yaml:10]: it seems stackage [lts-20.8](https://www.stackage.org/lts-20.8) supports ghc 9.2.5, so why not

```suggestion
resolver: ghc-9.2.5
```
?

**Review by akrmn — APPROVED** [review]
LGTM other than question about resolver version

File: stack-exact.yaml (hunk 1 above)
Comment (shayne-fletcher) [reply stack-exact.yaml:10]: i chose 9.2.2 because its the minimum supported build compiler for ghc-9.6.1 flavored builds.

**Comment by shayne-fletcher** [comment]
looks like we got a bit hung up on windows da-8.81 @akrmn. hopefully it's transient!

**Comment by garyverhaegen-da** [comment]
> looks like we got a bit hung up on windows da-8.81 @akrmn. hopefully it's transient!

Restarted. 🤞

File: stack-exact.yaml (hunk 1 above)
Comment (akrmn) [reply stack-exact.yaml:10]: gotcha! thanks for the explanation

**Pull request merged**